app/controllers/mixins/ems_common_angular.rb

```diff
@@ -420,6 +423,12 @@ def set_ems_record_vars(ems, mode = nil)
       build_connection(ems, endpoints, mode)
     end
 
+    def get_hostname_from_routes(ems, hostname, port, token)
+      client = ems.class.raw_connect(hostname, port, :bearer => token)
```

Comment:
@yaacov @nimrodshn why going to hawkular? We should have this information in our db.